pkg/config/config_template.yaml

```diff
@@ -1615,6 +1615,40 @@ api_key:
   #   - HISTSIZE
   #   - HISTFILESIZE
 
+  ## @param activity_dump - custom object - optional
+  ## Activity dumps section to configure if/how the agent sends activity dumps to the backend for security profiles learning.
+  #
+  # activity_dump:
+
+    ## @param enabled - boolean - optional - default: false
+    ## @env DD_RUNTIME_SECURITY_CONFIG_ACTIVITY_DUMP_ENABLED - boolean - optional - default: false
+    ## Set to true to activate the security profiles feature.
+    #
+    #  enabled: false
+
+    ## @param traced_cgroups_count - integer - optional - default: 5
+    ## @env DD_RUNTIME_SECURITY_CONFIG_ACTIVITY_DUMP_TRACED_CGROUPS_COUNT - integer - optional - default: 5
+    ## Defines the number of concurrent cgroups to be learned.
```

Comment:
nit: Is there an alternative word to use for `learned` here and in other options? I don't know the feature enough to know how it's used to recommend an alternative though.

Reply:
Hmm, `learned` lgtm, but we can replace it with `traced` I think. I'll do the change